subprojects/docs/src/docs/release/notes.md

```diff
@@ -40,6 +40,13 @@ See the User Manual section on the “[Feature Lifecycle](userguide/feature_life
 
 The following are the features that have been promoted in this Gradle release.
 
+### Gradle Dependency Cache can be relocated
+
+With this release, the Gradle Dependency cache, that is the content under `$GRADLE_HOME/caches/modules-2`, can be fully relocated.
```

Comment:
That's not quite correct. Only the `metadata-90` directory is. Previous versions are _not_

Reply:
That's true, but I wonder if we need to be _that_ specific.